### PR TITLE
Release notes for 5.6.42-82.2

### PR DESCRIPTION
**Pull request by borys-belinsky**

### CODE REVIEW

File: doc/source/release-notes/Percona-Server-5.6.42-84.2.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.. rn:: 5.6.42-84.2
+
+================================================================================
+|Percona Server| |release|
+================================================================================
+
+Percona is glad to announce the release of |Percona Server| |release| on
+November 29, 2018 (Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.40-84.0/>`_
```
Comment (hrvojem) [inline doc/source/release-notes/Percona-Server-5.6.42-84.2.rst]: Wrong link/version

**Review by hrvojem — CHANGES_REQUESTED** [review]

File: doc/source/release-notes/Percona-Server-5.6.42-84.2.rst [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+.. rubric:: Other improvements
+
+- :psbug:`4881`: Add LLVM/clang 7 to Travis-CI
+
+
+Bug Fixed
+================================================================================
+
+- Slave replication could break if upstream bug :mysqlbug:`74145`
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.6.42-84.2.rst:31]: Need at least some details of 74145

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: doc/source/release-notes/Percona-Server-5.6.42-84.2.rst (hunk 1 above)
Comment (borys-belinsky) [reply doc/source/release-notes/Percona-Server-5.6.42-84.2.rst]: Link corrected to refer to 5.6.42-84.2

File: doc/source/release-notes/Percona-Server-5.6.42-84.2.rst (hunk 2 above)
Comment (borys-belinsky) [reply doc/source/release-notes/Percona-Server-5.6.42-84.2.rst:31]: Add the description after the number: FLUSH LOGS improperly disables the logging if the log file cannot be accessed

**Review by laurynas-biveinis — APPROVED** [review]

**Review by hrvojem — APPROVED** [review]
LGTM

**Pull request merged**